.ci/install_kata_kernel.sh

```diff
@@ -65,9 +65,10 @@ get_packaged_kernel_version() {
 }
 
 build_and_install_kernel() {
+	# Always build and install the kernel version found locally
 	info "Install kernel from sources"
 	pushd "${tmp_dir}" >> /dev/null
-	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup"
+	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup" -v "${kernel_version}"
 	"${kernel_repo_dir}/kernel/build-kernel.sh" "build"
```

Comment:
I'd add the same to build and install as needs to find the kernel path based on the kernel version.